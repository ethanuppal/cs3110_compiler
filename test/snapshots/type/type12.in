func foo(a: Int) {
    print a
}
func bar(a: Int, b: Bool) {
    print a
    print b
}
func baz(a: Bool, b: Int, c: Int*) {
    print a
    print b
    print c
}
