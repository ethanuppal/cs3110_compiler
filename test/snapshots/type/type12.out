func foo(a: Int) -> Unit {
    print a
}
func bar(a: Int, b: Bool) -> Unit {
    print a
    print b
}
func baz(a: Bool, b: Int, c: Int*) -> Unit {
    print a
    print b
    print c
}
